probabilistic.go

```diff
@@ -0,0 +1,1142 @@
+package redis
+
+import (
+	"context"
+	"fmt"
+	"github.com/redis/go-redis/v9/internal/proto"
+)
+
+type ProbabilisticCmdable interface {
+	BFAdd(ctx context.Context, key, element interface{}) *BoolCmd
+	BFCard(ctx context.Context, key string) *IntCmd
+	BFExists(ctx context.Context, key, element interface{}) *BoolCmd
```

Comment:
`key string`?

Reply:
Ah, missed this when I converted "element" to interface (it used to be a string). Thanks for catching it.